types/fetch-mock/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "strict-export-declare-modifiers": false
```

Comment:
Can you fix this fallout instead of suppressing the lint? Seems like you'd need to add a few `export`s.